django/contrib/postgres/aggregates/mixins.py

```diff
@@ -3,18 +3,22 @@
 
 class OrderableAggMixin:
     def __init__(self, *expressions, ordering=(), **extra):
-        if isinstance(ordering, (list, tuple)):
-            self.order_by = OrderByList(*ordering)
+        if ordering:
```

Comment:
Nit: Could reduce some indentation here by making the first branch `if not ordering:` 🤔

Reply:
Changed :+1: